x/bank/types/errors.go

```diff
@@ -6,9 +6,9 @@ import (
 
 // x/bank module sentinel errors
 var (
-	ErrNoInputs              = sdkerrors.Register(ModuleName, 2, "no inputs to send transaction")
-	ErrNoOutputs             = sdkerrors.Register(ModuleName, 3, "no outputs to send transaction")
-	ErrInputOutputMismatch   = sdkerrors.Register(ModuleName, 4, "sum inputs != sum outputs")
+	// ErrNoInputs              = sdkerrors.Register(ModuleName, 2, "no inputs to send transaction")
+	// ErrNoOutputs             = sdkerrors.Register(ModuleName, 3, "no outputs to send transaction")
+	// ErrInputOutputMismatch   = sdkerrors.Register(ModuleName, 4, "sum inputs != sum outputs")
```

Comment:
Can we remove these?

Reply:
I kept them as a way to reserve the error codes so they don't get accidentally recycled.  I can just switch it to a comment saying the error codes are reserved?

Reply:
Yeah I was gonna suggest that 👍